src/process_utility.c

```diff
@@ -2667,7 +2667,7 @@ process_index_start(ProcessUtilityArgs *args)
 	hcache = ts_hypertable_cache_pin();
 	ht = ts_hypertable_cache_get_entry_rv(hcache, stmt->relation);
 
-	if (NULL == ht)
+	if (!ht)
```

Comment:
nit: I think these changes as well as https://github.com/timescale/timescaledb/pull/5252/files#diff-038649a1ce33b21f89e080d566755f896778f29c2ae7c5d9a1bf6e7975b5f48cR1409 could be made separately in a different PR, if they are necessary?